test/benchmarks/package.json

```diff
@@ -3,14 +3,17 @@
   "version": "0.0.0",
   "description": "benchmarks for LevelUP",
   "main": "index.js",
+  "scripts": {
+    "postinstall": "cd ../../ && npm install"
```

Comment:
The assumption was that the "root" is already installed (and that should be documented).

This `postinstall` step hurts flexibility (e.g. when you have linked packages), can you please remove it?

Reply:
you can update document to fix it.

Reply:
Correct. So the steps are:

```
npm i
cd test/benchmarks
npm i
node .
```